src/main/webapp/WEB-INF/views/home/homeNotSignedIn.html

```diff
@@ -1,27 +1,135 @@
 <!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org">
+<!-- <link rel="stylesheet" type="text/css" href="bootstrap.min.css"></link>
+<link rel="stylesheet" type="text/css" href="myStyle.css"></link>
+<meta charset="UTF-8"></meta>-->
+<!-- <link href='http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,700,300,600,800,400&subset=latin,latin-ext'
+      rel='stylesheet' type='text/css'>
+-->
 <head>
-    <title>Welcome to the Spring MVC Quickstart application! Get started quickly by signing up!</title>
+    <title th:text="#{view.index.title}">Welcome!</title>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-    <link href="../../../resources/css/styles.css" rel="stylesheet" media="screen" th:href="@{/resources/css/styles.css}"/>
-    <link href="../../../resources/css/core.css" rel="stylesheet" media="screen" th:href="@{/resources/css/core.css}" />
+    <link href="../../../resources/css/myStyle.css" rel="stylesheet" media="screen" th:href="@{/resources/css/myStyle.css}"/>
+    <link href="../../../resources/css/bootstrap.min.css" rel="stylesheet" media="screen" th:href="@{/resources/css/bootstrap.min.css}"/>
+    <!--<link href="../../../resources/css/core.css" rel="stylesheet" media="screen" th:href="@{/resources/css/core.css}" />-->
     <script src="http://code.jquery.com/jquery-latest.js"></script>
     <script src="../../../resources/js/bootstrap.min.js" th:src="@{/resources/js/bootstrap.min.js}"></script>
 </head>
 <body>
-<div th:replace="fragments/header :: header">Header</div>
+
 <div class="container">
-    <div class="text-center">
+    <nav class="navbar navbar-default navbar-fixed-top">
```

Comment:
this should be added in the header fragment and <div th:replace="fragments/header :: header">Header</div> above should not be removed